src/main/java/fi/aalto/cs/apluscourses/presentation/ideactivities/TutorialViewModel.java

```diff
@@ -91,4 +98,16 @@ public void cancelTutorial() {
   public @NotNull Tutorial getTutorial() {
     return tutorialExercise.getTutorial();
   }
+
+  public int getCurrentTaskIndex() {
+    return currentTaskIndex;
+  }
+
+  private void incrementIndex() {
```

Comment:
I suggest renaming this to `incrementTaskIndex` to be consistent with `getCurrentTaskIndex`